test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxserviceTests.cs

```diff
@@ -532,6 +537,78 @@ public async void GenerateDeployScriptWithSqlCmdVariables()
+            dacfxRequestContext.Setup((RequestContext<DacFxOptionsResult> x) => x.SendResult(It.Is<DacFxOptionsResult>((result) => ValidateOptions(expectedResults, result.DeploymentOptions) == true))).Returns(Task.FromResult(new object()));
+
+            DacFxService service = new DacFxService();
+            string file = Path.Combine(publishProfileFolder, "profileWithOptions.publish.xml");
+
+            var getOptionsFromProfileParams = new GetOptionsFromProfileParams
+            {
+                ProfilePath = file
+            };
+
+            await service.HandleGetOptionsFromProfileRequest(getOptionsFromProfileParams, dacfxRequestContext.Object);
+            dacfxRequestContext.VerifyAll();
+        }
+
+        // <summary>
+        /// Verify that default options are returned if a profile doesn't specify any options
+        /// </summary>
+        [Fact]
+        public async void GetOptionsFromProfileWithoutOptions()
+        {
+            DeploymentOptions expectedResults = new DeploymentOptions();
+            expectedResults.ExcludeObjectTypes = null;
+
+            var dacfxRequestContext = new Mock<RequestContext<DacFxOptionsResult>>();
+            dacfxRequestContext.Setup((RequestContext<DacFxOptionsResult> x) => x.SendResult(It.Is<DacFxOptionsResult>((result) => ValidateOptions(expectedResults, result.DeploymentOptions) == true))).Returns(Task.FromResult(new object()));
+
+            DacFxService service = new DacFxService();
+            string file = Path.Combine(publishProfileFolder, "profileNoOptions.publish.xml");
+
+            var getOptionsFromProfileParams = new GetOptionsFromProfileParams
+            {
+                ProfilePath = file
+            };
+
+            await service.HandleGetOptionsFromProfileRequest(getOptionsFromProfileParams, dacfxRequestContext.Object);
+            dacfxRequestContext.VerifyAll();
+        }
+
+        private bool ValidateOptions(DeploymentOptions expected, DeploymentOptions actual)
+        {
+            System.Reflection.PropertyInfo[] deploymentOptionsProperties = expected.GetType().GetProperties();
+            foreach (var v in deploymentOptionsProperties)
+            {
+                var defaultP = v.GetValue(expected);
+                var actualP = v.GetValue(actual);
+
+                if (v.Name == "ExcludeObjectTypes")
+                {
+                    Assert.True((defaultP == null && actualP == null) || ((defaultP as ObjectType[]).Length == (actualP as ObjectType[]).Length), "Number of excluded objects is different not equal");
+                }
+                else
+                {
+                    Assert.True((defaultP == null && actualP == null) || (defaultP == null && (actualP as string) == string.Empty) || defaultP.Equals(actualP), $"Actual Property from Service is not equal to default property for { v.Name}, Actual value: {actualP} and Default value: {defaultP}");
```

Comment:
extra space before `v.name`

Reply:
Github, when can I leave comments on specific chunks of text rather than full lines?!